magenta/models/music_vae/js/demo/index.ts

```diff
@@ -1,57 +1,63 @@
 import { MusicVAE, intsToBits, bitsToInts, intsToOneHot } from '../index';
 import * as dl from 'deeplearn'
 
-async function initializeDrums(){
-
-  const mvae:MusicVAE = await new MusicVAE('https://storage.googleapis.com/download.magenta.tensorflow.org/models/music_vae/dljs/drums_small_hikl').initialize();
+async function runDrums(){
+  const mvae:MusicVAE = await new MusicVAE(
```

Comment:
await new is weird, do you need the await here?

Reply:
moved initialize() to separate line for clarity